README.md

```diff
@@ -37,6 +37,209 @@
 * Can work with Extensions
 * Can work with making a Class and Enums
 
+## JF NEW TEXT
+Just as in any object-oriented programming language, classes in Swift encapsulate data and functionality. In practical terms, that means properties that are variables (`var`) and constants (`let`) as well as methods. (Class methods are functions that are declared within a class. Functions can also be declared outside of classes.)
+
+>For more on classes insert Xref to Classes
+
+## Create a New Class - Person
+You can create a Person class.
+
+Encapsulated in that class will be three properties: name, height, and favoriteFood. Here are the property declarations:
+
+		var name: String
+		var height: Double
+		var favoriteFood: String
+
```

Comment:
A prettier way this can be represented in markdown (referring to the code snippet), is doing so like this:

![markdown](https://i.imgur.com/jSvId5N.png)

Doing so will produce the following outcome, notice the colors!

```swift
var name: String
var height: Double
var favoriteFood: String
```